lib/matplotlib/axes/_axes.py

```diff
@@ -2648,6 +2648,9 @@ def bar_label(self, container, labels=None, *, fmt="%g", label_type="edge",
             When *fmt* is a string and can be interpreted in both formats,
             %-style takes precedence over {}-style.
 
+            .. versionadded:: 3.7
+               support for {}-style format string and callables
```

Comment:
```suggestion
               Support for {}-style format string and callables.
```
I think we want to start with a capital letter and end with a period, even if it's not a full sentence - like we do for parameter descriptions.